app/routes/index.tsx

```diff
@@ -82,66 +130,49 @@ export default function App() {
     return result.questions
   }
 
+  const {handleDragOver, handleDragStart, handleDragEnd, DragPlaceholder} =
+    useDraggable(moveQuestion)
+
   return (
     <>
-      <header className={minLogo ? 'min-logo' : 'fun-logo'}>
-        {minLogo ? (
-          <div className="logo-intro-group">
-            <Link to="/" onClick={(e) => reset(e)}>
-              <img className="logo" alt="logo" src={logoMinSvg} />
-            </Link>
-            <div className="intro">
-              Answering questions about
-              <h1>AI Safety</h1>
-            </div>
-          </div>
-        ) : (
-          <div className="logo-intro-group">
-            <Link to="/" onClick={(e) => reset(e)}>
-              <img className="logo" alt="logo" src={logoFunSvg} />
-            </Link>
-            <div className="intro">
-              <h1>
-                Welcome to <span className="highlight">stampy.ai</span>!
-              </h1>
-              I can answer questions about artificial general intelligence safety
-            </div>
-          </div>
-        )}
-        <div className="icon-link-group">
-          <CopyLink>
-            <Share />
-            Share link
-          </CopyLink>
-          <a href="https://get_involved.aisafety.info" className="icon-link">
-            <Users />
-            Get Involved
-          </a>
-          <a href="https://github.com/StampyAI/stampy-ui" className="icon-link">
-            <Code />
-            Help Code
-          </a>
-        </div>
-      </header>
+      <Header reset={reset} />
       <main onClick={handleSpecialLinks}>
         <Search
           onSiteAnswersRef={onSiteAnswersRef}
           openQuestionTitles={openQuestionTitles}
           onSelect={selectQuestion}
         />
-        <InfiniteScroll className="articles-container" fetchMore={fetchMoreQuestions}>
+
+        {/* Add an extra, draggable div here, so that questions can be moved to the top of the list */}
+        <div draggable onDragOver={handleDragOver({pageid: TOP})}>
+          &nbsp;
+        </div>
+        <DragPlaceholder pageid={TOP} />
+
+        <InfiniteScroll
+          className="articles-container"
+          fetchMore={fetchMoreQuestions}
+          onDragOver={handleDragOver()}
+        >
           {questions.map((question) => (
-            <Question
-              key={question.pageid}
-              questionProps={question}
-              onLazyLoadQuestion={onLazyLoadQuestion}
-              onToggle={toggleQuestion}
-            />
+            <Fragment key={question.pageid}>
```

Comment:
I considered having an additional element to wrap this and the drag placeholder, but it kept eating my event handlers, so I decided this was good enough

Reply:
cool